packages/stream-caip10-link/src/caip10-link.ts

```diff
@@ -159,19 +172,21 @@ export class Caip10Link extends Stream {
   }
 
   /**
-   * Makes the genesis commit from a given CAIP-10 AccountID
+   * Makes the genesis commit from a given CAIP-10 AccountId
    * @param accountId
    */
-  static makeGenesis(accountId: AccountID): GenesisCommit {
+  static makeGenesis(accountId: AccountId): GenesisCommit {
     // Ethereum addresses specifically are sometimes encoded with mixed case and
     // sometimes all lower case. In order to deal with this and not have different
     // links for different addresses we convert the address to lowercase.
     if (accountId.chainId.namespace === 'eip155') {
       accountId.address = accountId.address.toLowerCase()
     }
+
+    const legacyAccountId = `${accountId.address}@${accountId.chainId.toString()}`
```

Comment:
please add a comment that we are continuing to represent links using the legacy caip10 format to preserve backwards compatibility with existing links